docs/services/blockstorage.rst

```diff
@@ -1,89 +1,187 @@
 .. _block_storage:
 
-Block Storage
+=============
+Block storage
 =============
 
-Commands for Rackspace Cloud Block Storage.
+This section lists all block storage commands supported by Rackspace CLI.
 
-Synopsis
+Commands
 --------
 
-::
+All block storage commands are based on this syntax::
 
    rack block-storage <resource> <action> [command flags]
 
-Commands
---------
+*Command flags* allow you to customize certain attributes of the command,
+such as ``--name`` to name a volume. Type ``rack block-storage <resourse> <action> --help``
+to bring up a list *command flags* specific to the command.
 
 ``volume``
 ~~~~~~~~~~
 
-Block Storage Volume operations
+Block storage volume commands use this syntax::
```

Comment:
We don't need 'Block storage' here since we're on the `Block Storage` page. I think just 'Volume commands use...'